Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/PhysicsPicking.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <Editor/Picking.h>
+
+namespace EMotionFX
+{
+    static constexpr float PickingMargin = 0.01f;
+
+    class PhysicsPicking : public Picking
```

Comment:
Question: is it worth adding an API comment for this class?

Reply:
Done as part of other changes (getting rid of the base class).